.github/workflows/github-actions-alpha-version.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           npm publish --registry https://registry.npmjs.org playbook-ui-${{ env.new_npm_alpha_version }}.tgz --tag alpha
       - name: Version Up, Distribute and Publish (RubyGems)
         run: |
-          gem build playbook_ui.gemspec
+          bin/build_gem
```

Comment:
Use bin stub instead of direct command so that we can keep the syntax clean here.